examples/step-69/step-69.cc

```diff
@@ -2734,57 +2781,101 @@ namespace Step69
     pcout << "MainLoop<dim>::output(t = " << t
           << ", checkpoint = " << checkpoint << ")" << std::endl;
 
-    TimerOutput::Scope scope(computing_timer, "main_loop - output");
-
-    if (checkpoint)
+    // If the asynchronous writeback option is set we launch a background
+    // thread performing all the slow IO to disc. In that case we have to
+    // make sure that the background thread actually finished running. If
+    // not, we have to wait to for it to finish because we would otherwise
+    // overwrite <code>output_vector</code>, rerun the
```

Comment:
just drop "because we would otherwise...". It disrupts the flow of the rest of the sentence as it's no longer clear what ", rerun" corresponds to: what we need to do, or what would otherwise happen.

(It's also the case that if the IO hasn't finished yet and you start another IO, you might eventually end up with *a lot* of threads still doing IO.)